### PR TITLE
Fix#1593:Add support for different file types

### PR DESCRIPTION
**Pull request by namish800**

### CODE REVIEW

File: apps/challenges/migrations/0001_initial.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ class Migration(migrations.Migration):
                 ('published', models.BooleanField(default=False, verbose_name='Publicly Available')),
                 ('enable_forum', models.BooleanField(default=True)),
                 ('anonymous_leaderboard', models.BooleanField(default=False)),
-                ('creator', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='challenge_creator', to='hosts.ChallengeHostTeam')),
+                ('creator', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE,
```
Comment (vkartik97) [inline apps/challenges/migrations/0001_initial.py]: I think the editor or automatic PEP8 formatter such as `autopep8` has made a lot of unnecessary changes which are not needed in this PR.

Comment (namish800) [reply apps/challenges/migrations/0001_initial.py]: Yes! I was following the steps given here https://evalai.readthedocs.io/en/latest/contribution.html

Comment (vkartik97) [reply apps/challenges/migrations/0001_initial.py]: But, `autopep8` has be applied to many files, even migrations!
Can you please try to resolve it using `Flake8 Linter` or do it manually as there are only three warnings:
```sh
./tests/unit/participants/test_views.py:606:25: E122 continuation line missing indentation or outdented
./tests/unit/participants/test_views.py:659:17: E122 continuation line missing indentation or outdented
./apps/jobs/views.py:176:16: E713 test for membership should be 'not in'
```
Let me know if you have questions :)

Comment (namish800) [reply apps/challenges/migrations/0001_initial.py]: I resolved these three warnings. Should I revert back the other files ?

Comment (vkartik97) [reply apps/challenges/migrations/0001_initial.py]: Just revert the files that are not required in this PR.

Comment (namish800) [reply apps/challenges/migrations/0001_initial.py]: Okay
